ppl/archive/finder.go

```diff
@@ -84,60 +82,46 @@ func AddPath(pathField string, absolutePath bool) FindOption {
 // be more efficient at large scale to allow multipe patterns that
 // are effectively OR-ed together so that there is locality of
 // access to the microindex files.
-func Find(ctx context.Context, zctx *resolver.Context, ark *Archive, query IndexQuery, hits chan<- *zng.Record, opts ...FindOption) error {
+func Find(ctx context.Context, ark *Archive, query index.Query, hits chan<- *zng.Record, opts ...FindOption) error {
 	opt := findOptions{
-		zctx:    zctx,
+		zctx:    resolver.NewContext(),
```

Comment:
Since this function returns `*zng.Record`'s , it should take a `zctx *resolver.Context` parameter that's used throughout.

Reply:
I was hoping to rid it of an extra arg, but having done a thorough looks at resolver.Context I can see you are right. Will fix.